WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -287,6 +297,7 @@ private extension ProductFormViewController {
     func editPriceSettings() {
         let priceSettingsViewController = ProductPriceSettingsViewController(product: product) { [weak self]
             (regularPrice, salePrice, dateOnSaleStart, dateOnSaleEnd, taxStatus, taxClass) in
+            ServiceLocator.analytics.track(.productPriceSettingsDoneButtonTapped)
```

Comment:
Just noting here that the “Done button tapped“ declaration is not entirely accurate. 

The block still gets called after pressing the **Back Button** and then **Save changes** in this dialog:

<img src="https://user-images.githubusercontent.com/198826/73091664-21b9d780-3e98-11ea-8a0b-9540c57c078f.png" width="240">

But I think this is what we want. 🙂

Reply:
ah great point! `product_price_settings_done_button_tapped` doesn't have the best wording here. does `product_*_settings_done_button_tapped_or_saved` or `product_*_settings_saved` sound better?

Reply:
I think it's better to rename all the events on `product_*_settings_saved`.
Since these features are not yet public, I would say that it is the ideal time to carry them out. I will open an issue regarding android. What do you think? Do you agree?

Reply:
`product_*_saved` sounds good to me 👍

Reply:
I agree with changing the event names as @pmusolino proposed in https://github.com/woocommerce/woocommerce-ios/pull/1766#issuecomment-578690455. 

Some minor corrections though to match the _start_ events. 

old name | new name
-- | --
product_name_done_button_tapped | **product_name_saved**
aztec_editor_done_button_tapped | **product_description_saved**
product_price_settings_done_button_tapped | product_price_settings_saved
product_shipping_settings_done_button_tapped | product_shipping_settings_saved
product_inventory_settings_done_button_tapped | product_inventory_settings_saved

This is close to nitpicking but I think it's best to keep the names consistent as much as possible. 😉 

Also, once this is merged, would you mind creating an issue for WCAndroid, @pmusolino?

Reply:
They sounds good! And sure, I'll open an issue on WCAndroid 💯